lib/runtime-c-api/src/lib.rs

```diff
@@ -164,8 +165,8 @@ pub unsafe extern "C" fn wasmer_validate(
     if wasm_bytes.is_null() {
         return false;
     }
-    let bytes: &[u8] =
-        unsafe { ::std::slice::from_raw_parts_mut(wasm_bytes, wasm_bytes_len as usize) };
+    let bytes: &[u8] = ::std::slice::from_raw_parts_mut(wasm_bytes, wasm_bytes_len as usize);
```

Comment:
A typical use of an `unsafe` block inside an `unsafe` function?

Reply:
Yes, these needed clean up after unsafe was added to the function declaration.